CHANGELOG.md

```diff
@@ -8,7 +8,9 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
-## Added
+## [1.2.1] - 2024-03-18
```

Comment:
You also have to update the end of this file to create the compare link with this new release

Reply:
Wooopsie, I forgot it